extensions/email/src/test/java/org/mule/extension/email/retriever/POP3STestCase.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.extension.email.retriever;
+
+import org.junit.Ignore;
+
+@Ignore
+public class POP3STestCase extends POP3TestCase
```

Comment:
instead of doing this, I would parameterize the above test and just have one that runs twice, each with different configs. Look at FTP/SFTP for inspiration